ngraph/core/include/ngraph/function.hpp

```diff
@@ -170,6 +170,16 @@ namespace ngraph
         /// \param result Result node to delete
         void remove_result(const std::shared_ptr<op::Result>& result);
 
+        /// \brief Add new Parameter nodes to the list. Method doesn't validate graph, it should be
+        /// done manually after all changes.
+        /// \param params new Parameter nodes
+        void add_parameters(const ParameterVector& params);
```

Comment:
From my point of view if we introduce such methods, we need to add more information about it and how user can use it. Because user can remove parameter and think that the graph is still valid, but it is not true.

Reply:
Extended comments, is it more clear now?